pkgs/development/libraries/mesa/default.nix

```diff
@@ -2,7 +2,7 @@
 , pkgconfig, intltool, autoreconfHook, substituteAll
 , file, expat, libdrm, xorg, wayland, wayland-protocols, openssl
 , llvmPackages, libffi, libomxil-bellagio, libva-minimal
-, libelf, libvdpau, valgrind-light, python2
+, libelf, libvdpau, valgrind-light, python2, python27Packages
```

Comment:
pass in either `python` or `pythonPackages`, not both.

Reply:
Can I reference `Mako` (why is that capitalized?) from `python2` instead of `python27Packages`?

Reply:
You can reference packages like this:  `python2.pkgs.Mako`.